README.md

```diff
@@ -92,6 +92,52 @@ gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys <last eight symbols of
 gpg --keyserver hkp://keyserver.ubuntu.com --send-keys <last eight symbols of gnupg keyId>
 ```
 
+### Stencil Server API
+
+For serving the protobuf descriptor set artifacts and their versions we use a Stencil Server.
```

Comment:
`The descriptors can be hosted behind an HTTP end point and served for for dynamic fetches through HTTP clients. Here's a pattern using http GET and PUT API that enables reading and updating the descriptors set behind an http API. This section deals with how to achieve this. `
WDYT?